xbmc/addons/binary-addons/AddonDll.cpp

```diff
@@ -461,13 +477,15 @@ bool CAddonDll::CheckAPIVersion(int type)
    * present.
    */
   if (kodiMinVersion > addonVersion ||
-      addonVersion > AddonVersion(kodi::addon::GetTypeVersion(type)))
-  {
-    CLog::Log(LOGERROR, "Add-on '%s' is using an incompatible API version for type '%s'. Kodi API min version = '%s', add-on API version '%s'",
-                            Name().c_str(),
-                            kodi::addon::GetTypeName(type),
-                            kodiMinVersion.asString().c_str(),
-                            addonVersion.asString().c_str());
+    addonMinVersion > AddonVersion(kodi::addon::GetTypeVersion(type)))
+  {
+    CLog::Log(LOGERROR, "Add-on '%s' is using an incompatible API version for type '%s'. Kodi API min version = '%s/%s', add-on API version '%s%s'",
+      Name().c_str(),
+      kodi::addon::GetTypeName(type),
+      kodi::addon::GetTypeVersion(type),
+      kodiMinVersion.asString().c_str(),
+      addonMinVersion.asString().c_str(),
+      addonVersion.asString().c_str());
```

Comment:
On newest CLog is no more a "c_str()" required and the %s if wished can be used mit {}.